packages/nocodb-sdk/src/lib/columnHelper/columns/Attachment.ts

```
@@ -0,0 +1,49 @@
+  columnDefaultMeta = {};
+
+  serializeValue(value: any, params: SerializerOrParserFnProps['params']) {
+    if (!value) return null;
+
+    let parsedVal = [];
+
+    try {
+      parsedVal = parseProp(value);
+
+      if (ncIsArray(parsedVal)) {
+        parsedVal = parsedVal;
```

Biome (1.9.4)

[error] parsedVal is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)